Android Payment Apps developer's guide update (#8676)
google/WebFundamentals: src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md

there’s no need for detailed error codes to be sent from the payment app to the
merchant website.

> Note: In Chrome 84 or later, if the activity result of a payment response
> received from the invoked payment app is set to `RESULT_OK`, then Chrome will
> check for non-empty `methodName` and `details` in its extras. If the validation
> fails Chrome will reject the `request.show()` promise with one of the following
> developer facing error messages:

Reviewer: ```suggestion
Note: In Chrome 84 and later, if the activity result of a payment response
received from the invoked payment app is `RESULT_OK`, then Chrome will
check for non-empty `methodName` and `details` in its extras. If the validation
fails Chrome will return a rejected promise from `request.show()` with one of the following
developer facing error messages:
```

I think this is more clear, but I'm not 100% sure.

Author: I think this is fine except the first `or`.

> <pre class="prettyprint devsite-code-highlight">
> 'Payment app returned invalid response. Missing field "details".'
> 'Payment app returned invalid response. Missing field "methodName".'
> </pre>

Reviewer: ```suggestion

<pre class="prettyprint devsite-code-highlight">
'Payment app returned invalid response. Missing field "details".'
'Payment app returned invalid response. Missing field "methodName".'
</pre>
```

Reviewer: Just added an extra line break.
